cli/src/pcluster/models/cluster.py

```diff
@@ -45,6 +46,8 @@
 
 # ---------------------- Storage ---------------------- #
 
+MAX_STORAGE_COUNT = {"ebs": 5, "efs": 1, "fsx": 1}
```

Comment:
Minor: how about moving this into `NumOfStorageValidator`

Reply:
Let's think about this, thanks. I'll go ahead for now.